ui/app/components/clients/current.js

```diff
@@ -1,22 +1,51 @@
 import Component from '@glimmer/component';
-
+import { tracked } from '@glimmer/tracking';
 export default class Current extends Component {
   chartLegend = [
     { key: 'entity_clients', label: 'entity clients' },
     { key: 'non_entity_clients', label: 'non-entity clients' },
   ];
+  @tracked selectedNamespace = null;
+
+  // TODO CMB pass in from parent
```

Comment:
Looks like this TODO is done? Would love to see some JSDocs for this component

Reply:
That comment was confusing, corrected. Waiting on the `version` endpoint and left this is a placeholder to test the conditionals. Can remove if that'd be easier?

This file is similar to `dashboard.js` and looks like this in the `index.hbs`
`  <Clients::Current @tab={{this.tab}} @model={{this.model}} @isLoading={{this.currentlyLoading}} />`